security/tor/src/opnsense/mvc/app/models/OPNsense/Tor/Relay.xml

```diff
@@ -10,6 +10,18 @@
       <Required>N</Required>
       <mask>/^(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}|[a-f0-9:]{2,})$/i</mask>
     </host>
+    <hostv6 type="TextField">
+      <Required>N</Required>
+      <mask>/^(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}|[a-f0-9:]{2,})$/i</mask>
+    </hostv6>
+    <outboundbind type="TextField">
+      <Required>N</Required>
+      <mask>/^(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}|[a-f0-9:]{2,})$/i</mask>
```

Comment:
``|[a-f0-9:]{2,}`` is for v6 and makes no sense here as well as v4 at the bottom